code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -107,25 +107,27 @@
 	else
 		return ..()
 
-/obj/item/reagent_containers/blood/attack(mob/M, mob/user, def_zone)
+/obj/item/reagent_containers/blood/attack(mob/living/carbon/C, mob/user, def_zone)
 	if(user.a_intent == INTENT_HELP && reagents.total_volume > 0)
```

Comment:
Probably add an `istype(C)` check here too.

Reply:
Why not iscarbon(C), wouldnt it do the same thing and be more readable?

Reply:
fine.